components/script/dom/document.rs

```diff
@@ -573,9 +573,11 @@ pub(crate) struct Document {
     #[conditional_malloc_size_of]
     custom_element_reaction_stack: Rc<CustomElementReactionStack>,
     #[no_trace]
-    #[ignore_malloc_size_of = "type from external crate"]
     /// <https://html.spec.whatwg.org/multipage/#active-sandboxing-flag-set>,
     active_sandboxing_flag_set: Cell<SandboxingFlagSet>,
+    #[no_trace]
+    /// <https://html.spec.whatwg.org/multipage/#active-sandboxing-flag-set>
```

Comment:
Shouldn't this be https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigation-supporting-concepts:sandboxing-flag-set instead? If that's the case, do consider if we eventually can make use of https://github.com/servo/servo/blob/739855ad947ec53a5f1e2fccc0891d515a10520c/components/script/dom/servoparser/mod.rs#L858-L869 and pass that along in these algorithms

Reply:
Ah I see below that you do use it. I guess I am confused why we now have `active_sandboxing_flag_set` and `creation_sandboxing_flag_set`. Can't we unify those?

Reply:
Ah, this is meant to be this: https://html.spec.whatwg.org/multipage/browsers.html#determining-the-creation-sandboxing-flags. According to https://html.spec.whatwg.org/multipage/browsing-the-web.html#snapshotting-target-snapshot-params, the targetSnapshotParams are supposed to refer to a snapshot of these creation parameters. Then https://html.spec.whatwg.org/multipage/browsing-the-web.html#snapshotting-target-snapshot-params says that they should be unioned with the ones from the headers as far as I can understand.

It makes a bit of sense too. If an `<iframe>` is crated with a certain set of sandbox parameters, those should always apply no matter what the CSP headers say.